shapeshift/web
# feat: nftApi initialState blacklist demo wallet spam NFTs

## Description

Does what it says on the box - adds a client-side blacklist of specific AssetIds for the native wallet spam NFTs. The first, safe heuristics towards tackling #5301.

Safe because this ensures we don't blacklist more than necessary as the blacklist is explicit. The downside to this safety is we will have to manually add new native spam NFT collections AssetIds, meaning this is only part of tackling #5301.

## Pull Request Type


## Issue (if applicable)



contributes to #5301

## Risk



- Ensure the blacklist is made of scam NFTs i.e we didn't include a non-spammy one there

## Testing



- Connect to native wallet and go to NFTs
- Ensure no scammy NFT is present there

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)

- Prod

![app shapeshift com_](https://github.com/shapeshift/web/assets/17035424/1cb2d6f8-a8f0-4967-89a6-774d5d5c2085)

- This diff

<img width="1712" alt="Screenshot 2023-09-18 at 10 28 37" src="https://github.com/shapeshift/web/assets/17035424/843d510d-8160-4ff8-82f3-2e91495d780f">
